server/data/data-library/src/main/java/org/apache/james/mailrepository/lib/AbstractMailRepository.java

```
@@ -141,6 +141,7 @@ public void remove(String key) throws MessagingException {
         if (lock(key)) {
             try {
                 internalRemove(key);
+            } catch (MessagingException e) {
```

Why do you ignore this exception?